collectors/APICalls/breakpoints.js

```diff
@@ -11,6 +11,9 @@ const breakpoints = [
             {name: 'sessionStorage'},
             {name: 'indexedDB'},
             {name: 'name'}, // can pottentially be used to link two tabs
+            {name: 'navigator.xr'},
```

Comment:
Why you decided to put it both here and under `Navigator ` object?

Reply:
I wasn't sure if calls to navigator.xr and window.navigator.xr would be caught the same way, but it's not consistent anyway and should probably just be in navigator.